solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/SolidityParsingPhase.java

```diff
@@ -0,0 +1,61 @@
+package org.sonarsource.solidity.frontend;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.Token;
+import org.antlr.v4.runtime.TokenStream;
+import org.sonarsource.solidity.frontend.SolidityParser.SourceUnitContext;
+
+public class SolidityParsingPhase {
+
+  // important to maintain order of comments for reporting
+  public final Set<Token> comments;
+  public Integer emptyLines;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make emptyLines a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=87&issues=AWUaV_Hd8anAXINB_Qkz&open=AWUaV_Hd8anAXINB_Qkz)

[ISSUE_KEY]: AWUaV_Hd8anAXINB_Qkz